lib/app.rb

```diff
@@ -0,0 +1,41 @@
+require '../lib/database_connection'
+
+require '../lib/order_repository'
+require '../lib/order'
+require '../lib/item_repository'
+require '../lib/item'
+
+DatabaseConnection.connect('shop_manager_library')
+
+orders = OrderRepository.new(Kernel)
+items = ItemRepository.new(Kernel)
+
+def interactive_menu
+    loop do
+        puts "What would you like to do?"
+        puts ("1. List all items")
+        puts ("2. create new item")
+        puts ("3. List all orders")
+        puts ("4. create new order")
+        puts ("9. Exit")
+        answer = gets.chomp
+        case answer
```

Comment:
having more than one method here could be useful to avoid having too many lines in one method